Connectors/Autocad/Speckle.Connectors.AutocadShared/HostApp/AutocadInstanceObjectManager.cs

```diff
@@ -50,7 +53,10 @@ public UnpackResult<AutocadRootObject> UnpackSelection(IEnumerable<AutocadRootOb
   private void UnpackInstance(BlockReference instance, int depth, Transaction transaction)
   {
     var instanceIdString = instance.Handle.Value.ToString();
-    var definitionId = instance.BlockTableRecord;
+    var hasAnonymousBlockTableRecordDefinition = instance.AnonymousBlockTableRecord != ObjectId.Null;
```

Comment:
deserves some explanation/note, what is the relationship between "anonymous" and "dynamic"